Networking/Networking/Remote/ProductReviewsRemote.swift

```diff
@@ -25,19 +25,27 @@ public final class ProductReviewsRemote: Remote, ProductReviewsRemoteProtocol {
     ///                determines fields present in response. Default is view.
     ///     - pageNumber: Number of page that should be retrieved.
     ///     - pageSize: Number of Orders to be retrieved per page.
+    ///     - products: Limit result set to reviews assigned to specific product IDs.
+    ///     - status: Limit result set to reviews assigned a specific status. If empty, the default is `all`.
```

Comment:
> If nil, the default is `all`

Could be a bit more accurate 😇

Reply:
You are right. I improved it here 4ca37984edb45a048b01cc8fcd5f9c872ca37076 👍